packages/stencil/src/components/datacom-breadcrumb/datacom-breadcrumb.css

```diff
@@ -0,0 +1,54 @@
+:host {
```

Comment:
Just went and ran your branch to see if everything looks okay, and it seems you're last link in a group of breadcrumb links still has a chevron at the end:
![image](https://user-images.githubusercontent.com/38770596/219225233-dcf40310-08cd-488b-bc9c-311dc50f75dc.png)

Should look to hide this with CSS. (You can achieve this the same way you've done with other components you've developed that had to deal with ":last-child"-ren).